tests/Silex/Tests/Provider/TwigServiceProviderTest.php

```diff
@@ -116,4 +116,13 @@ public function testFormWithoutCsrf()
 
         $this->assertInstanceOf('Twig_Environment', $app['twig']);
     }
+
+    public function testServices()
+    {
+        $app = new Application();
+        $app->register(new TwigServiceProvider());
+        foreach ($app->keys() as $id) {
+            $app[$id];
```

Comment:
The test is here only to check that the different services can be created without errors. If you execute the test on the previous version the test is broken. I think it's not a good practice to be able to create services even if the services are instantiated on-demand. 
In my codebase we check all the services can be created and since the last stable version is does not work anymore.